### DISCUSSION TITLE
Yoctopuce Yocto-VOC-V3 with Artisan for tracking humidity at the exhaust

### DISCUSSION THREAD

**Discussion by McPatPyhton** [artisan-roaster-scope/artisan#1534] [General]
Hello,

I would like to track the humidity at the exhaust port of my roaster and considering to track in Artisan with Yoctopuce Yocto-VOC-V3 (https://www.yoctopuce.com/EN/products/usb-environmental-sensors/yocto-voc-v3) . now, i noted that the device is not directly supported but Artisan propose the Yocto Sensor (https://artisan-scope.org/devices/yoctopuce/) Now,  can i add the VOC-V3 with the Yocto Sensor addition in Extra Devices ?  And if so, how can i access the Humidity Sensor, one of the 4 sensors proposed by the VOC-V3?  Note that i am running with Artisan 2.8.2 Thanks , Patrick

**Comment by darrenaddy** [comment]
" the relative humidity scale is essentially useless or even misleading at
temperatures above 100°C."

Source:
https://macinstruments.com/blog/whats-wrong-with-relative-humidity-above-100-celsius/

Respectfully,
Darren Addy
Kearney, Nebraska

**Reply by McPatPyhton** [reply]
Hello,

Thank you for your reply, i will precise i will not attempt to put the VOC-V3 directly at the exhaust port, but near enough. So, to avoid exceeding the limit of the VOC V3 and what you mention.

Regards,
Patrick

**Comment by darrenaddy** [comment]
Just to be clear, I wasn't talking about exceeding the limit of the VOC V3.
I was talking about the folly of attempting to measure relative humidity of
air that is over 100C.

Respectfully,
Darren Addy
Kearney, NE

**Reply by McPatPyhton** [reply]
Hello Darren,

Understood, for information the idea that i am planning to adapt for my roaster is based on this idea
https://www.home-barista.com/roasting/humidity-sensor-on-exhaust-other-gas-sensors-t87191.html

Regards,
Patrick

**Comment by MAKOMO** [comment]
I haven't tested this for lack of hardware, but the Artisan YoctoSensor device type _[binds its two channels to any Yoctopuce sensor interface (YSensor) not yet allocated](https://artisan-scope.org/devices/yoctopuce/)_. Thus you need to add several of those YoctoSensor devices to bind to all published sensors of that module and just deactivate/hide those you are not interested in. Artisan does not provide a UI to specify the exact channel you want `YoctoSensor` to bind too. However, at anytime you can write script that reads data from any source you are interested in and run it as `External Program` to the ET/BT tab of the Device dialog.

**Reply by McPatPyhton** [reply]
Hello,

Thank you for the information, i will check the YoctoSensor the functionality when I receive the device. I currently use the version 2.8.2

Regards,
Patrick
